### ISSUE TITLE
Silent failure with some libraries.

### ISSUE THREAD

**Issue by ASavchenkov** [Shougo/neoinclude.vim#25]
maybe I'm doing something wrong (Although I have no way to know at the moment because the "EXAMPLES" portion of the documentation is kind of... blank.)

As far as I've surmised, what I'm meant to do is:

let g:neoinclude#paths = {'c++' : '/path/to/include/directory'}

which I have done. I'm not getting completions for that library however. Interestingly enough the same path given to syntastic seems to be fine. Is there something else I'm meant to do to get this working? If there is, would it be possible to update neoinclude.txt with such an example for those of us less adept with vim?

**Comment by Shougo** [comment]
```vim
let g:neoinclude#paths = {'cpp' : '/path/to/include/directory'}
```

>  If there is, would it be possible to update neoinclude.txt with such an example for those of us less adept with vim?

You can update the documentation!
Unfortunately, I have not much time.

**Comment by ASavchenkov** [comment]
I would be glad to, but I myself don't understand how to use it. It looks like I have set the right path, but it's just not doing the thing it should be. I suspect this is because I'm doing it wrong, but I don't have the prerequisite vimscript and plugin knowledge to figure out what the plugin is doing based on reading the source, hence why I commented on docs.

**Comment by ASavchenkov** [comment]
Interesting. I decided to get off of my lazy butt and do some testing.

Some libraries are fine, while some other's aren't. For example, neoinclude dealt with my homebrew installation of FLANN perfectly fine, however when I pointed it to my tensorflow installation it didn't even recognize the namespace. The tensorflow library also gives syntastic some trouble too, however syntastic simply takes about 2 seconds to actually get through it all while neoinclude seems to just quitely fail.

the differences between the two that I have found so far are that tensorflow include directory is 30 times larger than the flann include directory, at 18 MB vs 660KB, and tensorflow was installed via pip.


I had thought I was configuring something incorrectly, while in reality I had not, and there was simply an issue with the one library I was using heavily. I'll continue testing this with other libraries to see if I can figure out what's going on.
